apps/api/src/common/util.spec.ts

```diff
@@ -26,21 +26,41 @@ describe('util', () => {
     expect(decrypted).toEqual(plaintext)
   })
 
+  it('should fail to encrypt and decrypt a string', () => {
+    const keyPair = createKeyPair()
+    const plainText = 'hello world'
+    const encrypted = encrypt(keyPair.publicKey, plainText)
+    const modifiedText = 'modified hello world'
+    const decrypted = decrypt(keyPair.privateKey, encrypted)
```

Comment:
We would like to decrypt the same string that you encrypted, but with a different private key. This would fail the test

Reply:
Okay, so if I create a second keyPair using the createKeyPair() method and then use its private key to decrypt the message which was encrypted using the first keypair, would that be okay?

Reply:
Yeap thats the plan